source/includes/_feature-importing.md

```diff
@@ -31,7 +31,21 @@ ds <- newDatasetFromFile("my.csv", name="my survey")
 # All three steps are handled within newDatasetFromFile
 ```
 
-POST a Dataset Entity to the datasets catalog.
+POST a Dataset Entity to the datasets catalog. The available initial attributes
+to create a new dataset entity are (all optional):
```

Comment:
_endpoint-datasets.md says:

```
The body must contain a "name", and additional parameters "description" and
"archived" are allowed. You can also include a Crunch Table in a "table" key,
as discussed in the Feature Guide. Sending any other attribute will return a 400 response.
```

That seems to contradict what is said here. Which is correct? (Just tell me; I'm already editing that file.)